.github/workflows/hax.yml

```diff
@@ -50,4 +50,5 @@ jobs:
       - name: 🏃🏻‍♀️ Run hax extraction
         run: |
           eval $(opam env)
-          ./proof.sh extract
+          python3 extract_to_fstar.py --kyber-reference
+          python3 extract_to_fstar.py --crate specs/kyber
```

Comment:
I would expect `./extract_to_fstar.py --crate hacspec_kyber`. `spec/kyber` is not a crate but the path to a crate.

Reply:
Renamed the argument to `crate-path`.